src/ImpactEvaluator.sol

```diff
@@ -28,31 +24,30 @@ contract ImpactEvaluator is AccessControl {
     constructor(address admin) {
         _grantRole(DEFAULT_ADMIN_ROLE, admin);
         _grantRole(EVALUATE_ROLE, admin);
-        advanceRound();
+        advanceRound(true);
     }
 
     receive() external payable {}
 
-    function advanceRound() private {
-        rounds.push();
-        Round storage round = rounds[rounds.length - 1];
+    function advanceRound(bool firstRound) private {
+        if (!firstRound) {
+            currentRoundIndex++;
+        }
```

Comment:
I find it a bit ugly that we need a boolean argument `firstRound`. I find it difficult to quickly understand what does `advanceRound(true)` and `advanceRound(false)` mean.

Proposed alternatives:

(1)
Start counting the rounds from 1, so that state initialization can call `currentRoundIndex++` too.

(2) 
Split `advanceRound` into two parts:
- `initialiseCurrentRound()` - most of the logic except `currentRoundIndex++`
- `advanceRound()` - calls `currentRoundIndex++` and then `initialiseCurrentRound()`
Then the state initialization can call `initialiseCurrentRound`

Reply:
great idea (2)!